src/module_wrap.cc

```diff
@@ -494,6 +493,59 @@ Maybe<uv_file> CheckFile(const URL& search,
   return Just(fd);
 }
 
+PackageConfig emptyPackage = { false, false, "" };
+std::unordered_map<std::string, PackageConfig> pjson_cache_;
```

Comment:
Style: `pjson_cache`, no trailing underscore.  `package_json_cache` is a better name, IMO.

Substance: should be a member of `Environment`.